src/bfs/README.md

```diff
@@ -61,7 +61,7 @@
 * You can get #of vertices and #of edges by running ingest\_edge\_list.
 * If '-s' is specified, the program uses system mmap instead of umap.
 * The interface to the umap runtime library configuration is controlled by environment variables, see [Umap Runtime Environment Variables](https://llnl-umap.readthedocs.io/en/develop/environment_variables.html).
-* This is a multi-threads (OpenMP) program. You can control the number of threads using the environment variable OMP\_NUM\_THREADS. You can also control OpenMP's schedule algorithm of the main BFS loop using OMP\_SCHEDULE environment variable.
+* This is a multi-threads (OpenMP) program. You can control the number of threads using the environment variable OMP\_NUM\_THREADS.
```

Comment:
Could you append to the comment that a static parallel for loop schedule is used?